### ISSUE TITLE
partially automation

### ISSUE THREAD

**Issue by gogainda** [eregon/truffleruby-gem-tracker#2]
I think it should be possible to partially automate population of gems.yml file by using Github search:
```
path:"/.github/workflows" filename:"*.yml" truffleruby-head
```

**Comment by eregon** [comment]
That indeed seems to give good results:
https://github.com/search?q=path%3A%22%2F.github%2Fworkflows%22+filename%3A%22*.yml%22+truffleruby-head

I'm not sure how automation would work? Using the GitHub API to get those search results and then generate entries in `gems.yml` for gems which aren't there yet?

**Comment by gogainda** [comment]
yes, we can leave current gems.yml and add entries there automatically  (we can start from adding only green tests)
